src/brogue/IO.c

```diff
@@ -808,6 +808,10 @@ void mainInputLoop() {
             if (playingBack) {
                 rogue.playbackMode = true;
                 executePlaybackInput(&theEvent);
+#ifdef ENABLE_PLAYBACK_SWITCH
+                if(playingBack && !rogue.playbackMode)
+                    initializeMenuButtons(&state, buttons);
+#endif
```

Comment:
What's the purpose of this code?

Reply:
The buttons need to be redrawn when I come out of switchToPlaying, but the buttonState isn't in that context. The other option is to pass the buttonState and buttons into switchToPlaying.

Reply:
Ok, I think we don't need to do that, but I would advise (since the playingBack check is redundant)

```suggestion
#ifdef ENABLE_PLAYBACK_SWITCH
                if (!rogue.playbackMode) {
                    initializeMenuButtons(&state, buttons);
                }
#endif
```

with an accompanying comment explaining that executePlaybackInput will set rogue.playbackMode to false if we switch to playback, and so we need to re-initialise the menu buttons as they depend on the playback mode.